setup.cfg

```diff
@@ -33,6 +33,7 @@ install_requires =
     aiohttp
     pyhocon
     pydantic>=2.0
+    safetensors
```

Comment:
If safetensors is only used in PT we should not make it general dependency

Reply:
cool, no problem, I just did some search and realize this "safetensors" also supports TF and NumPy, 
maybe we do want to utilize it for general purpose later.